### ISSUE TITLE
Using jimgui with modules

### ISSUE THREAD

**Issue by mudlee** [ice1000/jimgui#51]
Hi!

Is there a way to use this great stuff with java modules? I include this dependency, the jar is there, but I see no possibility to require it in my `module-info.java`. Any idea?
```java
<dependency>
    <groupId>org.ice1000.jimgui</groupId>
    <artifactId>core</artifactId>
    <version>v0.9</version>
</dependency>
```

**Comment by ice1000** [comment]
I don't know how to package a java module from a gradle project. Can you help me?

**Comment by mudlee** [comment]
Sorry, I have no experienc with gradle, but it should not be that different. Maybe [this](https://guides.gradle.org/building-java-9-modules/) can help.

It's a kind of feature proposal.

**Comment by ice1000** [comment]
I got a dependency, say, jetbrains annotations. Does that library support j9 modules?

**Comment by ice1000** [comment]
Fixed in #56
